bsp/esp32_p4_function_ev_board/esp32_p4_function_ev_board.c

```diff
@@ -348,15 +400,20 @@ static lv_display_t *bsp_display_lcd_init(const bsp_display_cfg_t *cfg)
         .control_handle = lcd_panels.control,
         .buffer_size = cfg->buffer_size,
         .double_buffer = cfg->double_buffer,
-        .hres = BSP_LCD_V_RES,
-        .vres = BSP_LCD_H_RES,
+        .hres = BSP_LCD_H_RES,
+        .vres = BSP_LCD_V_RES,
         .monochrome = false,
         /* Rotation values must be same as used in esp_lcd for initial settings of the screen */
         .rotation = {
             .swap_xy = false,
             .mirror_x = true,
             .mirror_y = true,
         },
+#if CONFIG_BSP_LCD_COLOR_FORMAT_RGB888
+        .color_format = LV_COLOR_FORMAT_RGB888,
+#else
+        .color_format = LV_COLOR_FORMAT_RGB565,
+#endif
```

Comment:
```suggestion
#if LVGL_VERSION_MAJOR >= 9
#if CONFIG_BSP_LCD_COLOR_FORMAT_RGB888
        .color_format = LV_COLOR_FORMAT_RGB888,
#else
        .color_format = LV_COLOR_FORMAT_RGB565,
#endif
#endif
```
These cannot be used when using LVGL v8.

Reply:
Fixed.